get_predictions.py

```diff
@@ -30,10 +31,15 @@ def extract_vision_model_value(response: str, key: str) -> int:
     :param key: The key (case-sensitive) whose corresponding value is to be extracted.
     :return: The numerical value associated with the key.
     """
-    # Parse the JSON string into a dictionary
-    data = json.loads(response)
-    # Access and return the value for the specified key
-    return data[key]
+    try:
+        # Only select the {"ROAD_QUALITY": N} part of the response
+        response = response.split("ROAD_QUALITY")[1]
+        # Extract the value of the key
+        value = ast.literal_eval(response.split(":")[1].split("}")[0].strip())
```

Comment:
question: is there a cleaner way to do this?

Reply:
we can interpret it as json, which will look cleaner, but will be less robust and have more confusing error messages